packages/terra-base/src/Base.scss

```diff
@@ -31,3 +31,40 @@ body {
 *::after {
   box-sizing: inherit;
 }
+
+a {
+  background-color: transparent; /* Remove the gray background on active links in IE 10. */
+  color: var(--terra-link-color, #0065a3);
+  outline: none;
+  text-decoration: underline;
+}
+
+// stylelint-disable selector-no-qualifying-type
+a:visited,
+a.is-visited {
+  color: var(--terra-link-visted-color, #0065a3);
+}
+
+a:hover,
+a.is-hovered {
+  color: var(--terra-link-hover-color, #004c76);
+}
+
+a:focus,
+a.is-focused {
+  background: transparent;
```

Comment:
This will need to be updated to allow theming. Update this to `var(--terra-link-focus-background-color, transparent);`

 In the cerner-consumer-theme PR, you have a themable variable named `--terra-link-focus-background-color`.

Reply:
[Updated](https://github.com/cerner/terra-core/pull/1171/commits/a9401c240610fba74ffaa99ce4e0cd4f5f1040b0#diff-1008bd5b69971980781839ebbefe2560R55).